localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -84,7 +85,7 @@ services:
       - shared-database
       - shared-redis
     healthcheck:
-      test: ["CMD", "wget", "--header=apollo-require-preflight: true", "http://localhost:3001/graphql?query=%7B__typename%7D", "-O", "/dev/null"]
+      test: ["CMD", "wget", "--spider", "http://localhost:3001/healthz"]
```

Comment:
duplicated change already made on main. Alternatively, could merge main into the base branch and I can merge the base branch in here.

Reply:
probably _should_ merge the main branch in but wasn't sure if someone else was normally handling that